### ISSUE TITLE
runtime.js is not installed

### ISSUE THREAD

**Issue by bmillwood** [janestreet/base#11]
cc @hhugo 
I think the idea is that `src/runtime.js` should end up in (e.g.) `~/.opam/system/lib/base/runtime.js`

Same goes for `core_kernel` and `bin_prot`, but I won't open those issues separately for now.

**Comment by diml** [comment]
Indeed, we need to fix this before the v1 release. Looking at this now
